This patch adds in test data which is included in the upstream git repository
but not in the crates.io release. It was generated from the v3.0.0 tag in
the upstream git repository.

diff -urN src/tests/toml-feature-not-string/Cargo.toml system-deps/src/tests/toml-feature-not-string/Cargo.toml
--- system-deps/src/tests/toml-feature-not-string/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-feature-not-string/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = { version = "1", feature = 2 }
diff -urN src/tests/toml-feature-versions/Cargo.toml system-deps/src/tests/toml-feature-versions/Cargo.toml
--- system-deps/src/tests/toml-feature-versions/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-feature-versions/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testdata = { version = "4", v5 = { version = "5" }, v6 = { version = "6" }}
diff -urN src/tests/toml-good/Cargo.toml system-deps/src/tests/toml-good/Cargo.toml
--- system-deps/src/tests/toml-good/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-good/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,4 @@
+[package.metadata.system-deps]
+testdata = "4"
+testlib = { version = "1", feature = "test-feature" }
+testmore = { version = "2", feature = "another-test-feature" }
diff -urN src/tests/toml-missing-key/Cargo.toml system-deps/src/tests/toml-missing-key/Cargo.toml
--- system-deps/src/tests/toml-missing-key/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-missing-key/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1 @@
+no-pkg-config-here = true
diff -urN src/tests/toml-not-table/Cargo.toml system-deps/src/tests/toml-not-table/Cargo.toml
--- system-deps/src/tests/toml-not-table/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-not-table/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata]
+system-deps = "not a table"
diff -urN src/tests/toml-optional/Cargo.toml system-deps/src/tests/toml-optional/Cargo.toml
--- system-deps/src/tests/toml-optional/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-optional/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,4 @@
+[package.metadata.system-deps]
+testlib = { version = "1.0", optional = true, v5 = { version = "5.0", name = "testlib-5.0", optional = false }}
+testmore = { version = "2", v3 = { version = "3.0", optional = true }}
+testbadger = { version = "1", optional = true }
\ No newline at end of file
diff -urN src/tests/toml-override-name/Cargo.toml system-deps/src/tests/toml-override-name/Cargo.toml
--- system-deps/src/tests/toml-override-name/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-override-name/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+test_lib = { name = "testlib", version = "1.0", v1_2 = { version = "1.2" } }
diff -urN src/tests/toml-two-libs/Cargo.toml system-deps/src/tests/toml-two-libs/Cargo.toml
--- system-deps/src/tests/toml-two-libs/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-two-libs/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,3 @@
+[package.metadata.system-deps]
+testlib = { version = "1" }
+testanotherlib = { version = "1" }
diff -urN src/tests/toml-unexpected-key/Cargo.toml system-deps/src/tests/toml-unexpected-key/Cargo.toml
--- system-deps/src/tests/toml-unexpected-key/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-unexpected-key/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = { version = "1", color = "blue" }
diff -urN src/tests/toml-version-in-table-not-string/Cargo.toml system-deps/src/tests/toml-version-in-table-not-string/Cargo.toml
--- system-deps/src/tests/toml-version-in-table-not-string/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-version-in-table-not-string/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = { version = 1 }
diff -urN src/tests/toml-version-missing/Cargo.toml system-deps/src/tests/toml-version-missing/Cargo.toml
--- system-deps/src/tests/toml-version-missing/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-version-missing/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = { feature = "test-feature" }
diff -urN src/tests/toml-version-names/Cargo.toml system-deps/src/tests/toml-version-names/Cargo.toml
--- system-deps/src/tests/toml-version-names/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-version-names/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = { version = "1.2", v2 = { version = "2.0", name = "testlib-2.0" }, v3 = { version = "3.0", name = "testlib-3.0" }}
\ No newline at end of file
diff -urN src/tests/toml-version-not-string/Cargo.toml system-deps/src/tests/toml-version-not-string/Cargo.toml
--- system-deps/src/tests/toml-version-not-string/Cargo.toml	1970-01-01 00:00:00.000000000 +0000
+++ system-deps/src/tests/toml-version-not-string/Cargo.toml	2022-04-14 19:12:20.707022555 +0000
@@ -0,0 +1,2 @@
+[package.metadata.system-deps]
+testlib = 1
